package.json

```diff
@@ -63,6 +63,6 @@
     "protractor": "~5.4.0",
```

Comment:
As you've added a `.prettierrc` file, you should also specify which version of prettier you use.